docs/user/firststeps.rst

```diff
@@ -32,6 +30,26 @@ First, create a project directory:
     $ mkdir -p ~/renku-projects/myproject
     $ cd ~/renku-projects/myproject
 
+Install ``renku`` and start the shell:
+
+.. code-block:: console
+
+    $ pipenv install --pre renku
```

Comment:
Should we recommend `renku` as core dependency or development (`--dev`) dependency for the user projects?

Reply:
You're right, technically should be a dev dependency but maybe that's an unnecessary complication at this stage?

Reply:
Let's hope that people will not question `--dev` options.